lib/utilities/detect-deprecated-config.js

```diff
@@ -0,0 +1,8 @@
+var path = require('path');
+var glob = require('glob');
+
+module.exports = function() {
+  var deprecatedConfigPath = path.join(__dirname, '../../../..', 'config', 'deploy', '*.js');
```

Comment:
Yeh, I'm not sure if there is a better way either mate.  That's actually the main reason I put it out there to you guys.

Will make the change to the `path.join`.